### ISSUE TITLE
Step sequencer typein issues

### ISSUE THREAD

**Issue by baconpaul** [surge-synthesizer/surge#7521]
From bscross on discord

Typing into any of the step values in step sec kicks the focus to the modulators section when finished. Ideally, I'd like it to return focus to the same step value that the typein was invoked from. Also, it doesn't seem to like to take fractional input, even though it gives it to you as output, but you can't always land on them with the arrow or shift arrow keys, so if I want 7/12, or how the control presents it, 7 twelfths, that doesn't seem doable.

**Comment by mkruselj** [comment]
B-but I do this at the end of ShowStepRMB()...

![image](https://github.com/surge-synthesizer/surge/assets/2393720/ad9358d3-cebc-43ee-837e-37323f70bee3)

Focus is supposed to be properly grabbed?

**Comment by mkruselj** [comment]
I have a fix for the fractional input but have no idea about the focus thing. @baconpaul any hints here, perhaps? Thanks!

**Comment by mkruselj** [comment]
Next nightly will fix the fractional input at least.

**Comment by baconpaul** [comment]
Hi! Sounds like a separate bug altogether. Can you open a new GitHub issue or join our discord and share

1. What's your OS, Daw, etc...
2. How did you install and what made it crash

Lots of people switch skins all the time so I am surprised it crashes for you. But we should have more info and it should not be in this issue.

Thanks
